src/extensions/uptime.py

```diff
@@ -0,0 +1,20 @@
+from datetime import datetime
+
+import arc
+
+start_time = datetime.now()
+
+plugin = arc.GatewayPlugin("Blockbot Uptime")
+
+
+@plugin.include
+@arc.slash_command("uptime", "Show formatted uptime of Blockbot")
+async def uptime(ctx):
+    up_time = datetime.now() - start_time
+    h, ms = divmod(up_time.seconds, 3600)
+    m, s = divmod(ms, 60)
+    await ctx.respond(f"Uptime: **{up_time.days} days, {h} hours, {m} minutes, and {s} seconds**.")

```

Comment:
If `h == 1`, would it be displayed as `1 hours` in the output? If so, do we want to correct this to `1 hour`?

Reply:
Same applies to the other values.

Reply:
something like this?


```
d = up_time.days
plurals = tuple('s' * (i!=1) for i in [d,h,m,s])
await ctx.respond(f"Uptime: **{d} day%s, {h} hour%s, {m} minute%s, and {s} second%s**." % plurals)
```